docs/userguide.rst

```diff
@@ -315,6 +315,34 @@ and push the created image to the DockerHub image registry:
 
     $ docker push johndoe/myenv
 
+**Supporting arbitrary user IDs**
+
+In the Docker container ecosystem, the processes run in the containers by
+default use the root user identity. However, this may not be secure. If you
```

Comment:
In the Docker container ecosystem, the processes in the containers run under the default ``root`` user identity.  

Otherwise it reads well I think :+1: